docs/payments/concepts/payment-rails.md

```diff
@@ -231,6 +287,52 @@ async function terminateRail(railId) {
 }
 ```
 
+### 6. Rail Updates and Their Impact
+
+When a payment rail is modified using `modifyRailPayment`, it's important to understand:
+
+1. **What changes**: Typically the payment rate (`paymentRate`) is updated
+2. **What doesn't change**: Proof obligations, verification schedule, and data commitments remain the same
```

Comment:
When you mean that data commitments stay the same, this means that there is no update in the data being stored?